pyServer/GuestFishWrapper.py

```diff
@@ -398,20 +400,28 @@ def execute(self, storageUrl):
                             if (len(pathPrefix) > 0) :
                                 self.rootLogger.info("Adding prefix to path: " + pathPrefix )
                         
+                        # choose manifest based on request clientType and osType
                         parentFolder = "/etc/azdis/"
-                        manifestFile = parentFolder + osType + os.sep + self.mode.lower()
+                        clientFolder = parentFolder + self.clientType.lower() + os.sep
+                        manifestFile = clientFolder + osType + os.sep + self.mode.lower()
                         if not os.path.isfile(manifestFile):
                             self.rootLogger.warning("Manifest file " + manifestFile + " could not be located.")
-                            if not os.path.isdir(parentFolder + osType):
+                            if not os.path.isdir(parentFolder + self.clientType.lower()):
+                                # bad clientType in request payload
+                                self.WriteToResultFile(operationOutFile, "No manifests exists for client '" + self.clientType + "', trying 'supportability' as client...")
+                                self.clientType = "supportability"
+                                clientFolder = parentFolder + self.clientType.lower() + os.sep
```

Comment:
This feels like it should be a constant, but I guess we don't really do constants in our Python code?

Reply:
@waynekuo - How about this? We can keep adding constants in the common class.